X/XGBoost/build_tarballs.jl

```diff
@@ -95,7 +95,7 @@ for cuda_version in versions_to_build, platform in platforms
     end
 
     augmented_platform = Platform(arch(platform), os(platform);
-        cuda=isnothing(cuda_version) ? "none" : CUDA.platform(cuda_version)
+        cuda=isnothing(cuda_version) ? nothing : CUDA.platform(cuda_version)
```

Comment:
Why this change? With `nothing`, *any* artifact is selected. I think you really want `"none"`.